intranet/apps/polls/models.py

```diff
@@ -94,6 +94,23 @@ def get_users_voted(self):
                 users = list(q.get_users_voted())
         return users
 
+    def get_num_eligible_voters(self):
+        users = []
+        if self.groups.all():
+            for g in self.groups.all():
+                if users:
+                    users = list(set(DjangoGroup.objects.get(name=g).user_set.all()) | set(users))
+                else:
+                    users = list(DjangoGroup.objects.get(name=g).user_set.all())
+            return len(users)
```

Comment:
I would return the `count()`  of a `users` queryset.

You could make this even more efficient with some complex queryset magic :smiley:

Reply:
I'm concerned about doing that because users might overlap between groups. I'm trying to use Q queries to do this more efficiently.